### DISCUSSION TITLE
Is there a Javascript equivalent of beautifulsoup4(Python)

### DISCUSSION THREAD

**Discussion by RemcoHalman** [EddieHubCommunity/support#325] [Q&A] [answered]
I'm Trying to create a website to show all links on a given url. Also to show if they are broken or not (SEO purpose). I could make a Flask API and just use VueJS on the front, but would be nice if i could keep it just in VueJS. 

Tried to get it working but no luck so far [this is the repo](https://github.com/RemcoHalman/Seo-Link-Checker)

I know there are sites doing this but wanted to make my own version 😄

![Schermafbeelding 2020-10-10 om 17 06 43](https://user-images.githubusercontent.com/25010775/95658484-fdcc1e00-0b1a-11eb-9cee-4bd80b0bba42.jpg)

**Comment by eddiejaoude** [comment, marked as answer]
I am not aware of anything, but I came across a javascript alternative to beautifulsoup, not sure if that is helpful https://github.com/chishui/JSSoup

**Comment by AllanRegush** [comment]
[cheerio.js](https://cheerio.js.org/) is another option.

**Reply by RemcoHalman** [reply]
@AllanRegush Looks good! Ill look in to it
